### PR TITLE
check toHezEthereumAddress

### LINKED ISSUE #651: Bug TX Transfer
When we access to see details of a sent transfer dated on current day, it's breaking the app

### PR DESCRIPTION
**Pull request by adrifdez**
Closes #651 

### What does this PR does?

Fix the problem on transactions details that crash the app when the returned data comes without data on the destination bjj or hezEthereumAddres 

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Respect code style and lint
- [ ] Update documentation (if needed)

### CODE REVIEW

File: src/views/shared/transaction-info/transaction-info.view.jsx [hunk 1]
```diff
@@ -91,13 +91,15 @@ function TransactionInfo({
...
             }}
-            to={{
-              subtitle:
-                txData.toHezEthereumAddress?.toLowerCase() ===
-                INTERNAL_ACCOUNT_ETH_ADDR.toLowerCase()
-                  ? getPartiallyHiddenHermezAddress(txData.toBjj || txData.toBJJ)
-                  : getPartiallyHiddenHermezAddress(txData.toHezEthereumAddress),
-            }}
+            to={
+              txData.toHezEthereumAddress && {
```
Comment (AlbertoElias) [inline src/views/shared/transaction-info/transaction-info.view.jsx:95]: And what if it's a tx to an internal account and it has a BJJ but no ethereum address?

Comment (adrifdez) [reply src/views/shared/transaction-info/transaction-info.view.jsx:95]: The last code wasn't covering that scenario so I assume that it's rare to happen. This fix is covering a corner case where those values aren't coming. In theory should be return at least the value that was submitted.
At least with this fix doesn't crash the app in your case either

Comment (AlbertoElias) [reply src/views/shared/transaction-info/transaction-info.view.jsx:95]: But with this change, internal transactions will stop showing the To field, and we can't allow that.

Comment (adrifdez) [reply src/views/shared/transaction-info/transaction-info.view.jsx:95]: we can show the same address if it's the known case and cover for the unexpected

File: src/views/shared/transaction-info/transaction-info.view.jsx [hunk 2]
```diff
@@ -42,6 +43,10 @@ function TransactionInfo({
     if (txData.state === TxState.Forged) {
       return { subtitle: TxStatus.Confirmed };
     }
+    console.log(txData);
```
Comment (AlbertoElias) [inline src/views/shared/transaction-info/transaction-info.view.jsx:46]: Leftover log

File: src/views/shared/transaction-info/transaction-info.view.jsx [hunk 3]
```diff
@@ -42,6 +43,10 @@ function TransactionInfo({
...
+    if (txData.errorCode) {
```
Comment (AlbertoElias) [inline src/views/shared/transaction-info/transaction-info.view.jsx:47]: This should probably go in a different PR

File: src/views/shared/transaction-info/transaction-info.view.jsx [hunk 4]
```diff
@@ -84,20 +89,24 @@ function TransactionInfo({
...
+            from={myHermezAddress}
+            to={
+              (txData.toHezEthereumAddress && {
+                subtitle:
+                  txData.toHezEthereumAddress.toLowerCase() ===
+                  INTERNAL_ACCOUNT_ETH_ADDR.toLowerCase()
+                    ? getPartiallyHiddenHermezAddress(txData.toBjj || txData.toBJJ)
+                    : getPartiallyHiddenHermezAddress(txData.toHezEthereumAddress),
+              }) ||
+              (txData.fromAccountIndex == txData.toAccountIndex && myHermezAddress)
```
Comment (AlbertoElias) [inline src/views/shared/transaction-info/transaction-info.view.jsx:108]: ===?

File: src/views/shared/transaction-info/transaction-info.view.jsx [hunk 5]
```diff
@@ -84,20 +89,24 @@ function TransactionInfo({
...
-            to={{
-              subtitle:
-                txData.toHezEthereumAddress?.toLowerCase() ===
-                INTERNAL_ACCOUNT_ETH_ADDR.toLowerCase()
-                  ? getPartiallyHiddenHermezAddress(txData.toBjj || txData.toBJJ)
-                  : getPartiallyHiddenHermezAddress(txData.toHezEthereumAddress),
-            }}
+            from={myHermezAddress}
+            to={
+              (txData.toHezEthereumAddress && {
```
Comment (AlbertoElias) [inline src/views/shared/transaction-info/transaction-info.view.jsx:101]: If we can, let's change this to `toBjj`

File: src/views/shared/transaction-info/transaction-info.view.jsx (hunk 3 above)
Comment (adrifdez) [reply src/views/shared/transaction-info/transaction-info.view.jsx]: yes

**Review by AlbertoElias — APPROVED** [review]
